packages/svelte2tsx/index.d.ts

```diff
@@ -55,7 +55,7 @@ export function svelte2tsx(
     }
 ): SvelteCompiledToTsx
 
-export interface EmitDtsConig {
+export interface EmitDtsConfig {
```

Comment:
This is technically a breaking change. I think we can export an interface with the old name and mark it as deprecated
```ts
/**@deprecated*/
export interface EmitDtsConig extends EmitDtsConfig {}
```